packages/taquito-ledger-signer/README.md

```diff
@@ -13,14 +26,13 @@ const transport = await TransportU2F.create();
 const ledgerSigner = new LedgerSigner(transport);
 ```
 
-The constructor of the `LedgerSigner` class can take three other parameters. If none are specified, the default values are used.
+The constructor of the `LedgerSigner` class takes three other optional parameters. If none are specified, the following default values are used:
 
-- path: **default value is "44'/1729'/0'/0'"**  
-  You can use as a parameter the `HDPathTemplate` which refers to `44'/1729'/${account}'/0'`. You have to specify what is the index of the account you want to use. Or you can also use a complete path as a parameter.  
-  _More details about paths below_
-- prompt: **default is true**  
+- `path`: **default value is "44'/1729'/0'/0'"**  
+  You can use as a parameter the `HDPathTemplate` which refers to `44'/1729'/${account}'/0'`. You have to specify what is the index of the account you want to use. Or you can also use a complete path as a parameter. More details about paths [here](https://tezostaquito.io/docs/ledger_signer#derivation-paths-hd-wallet--bip-standards)
```

Comment:
specify what is the index ->  specify the index

Reply:
ah this was leftover from the previous doc, will amend